packages/core/core/src/applyRuntimes.js

```diff
@@ -66,8 +69,23 @@ export default async function applyRuntimes({
         if (applied) {
           let runtimeAssets = Array.isArray(applied) ? applied : [applied];
           for (let {code, dependency, filePath, isEntry} of runtimeAssets) {
+            let sourceName = path.join(
+              path.dirname(filePath),
+              `runtime-${md5FromString(code)}.${bundle.type}`,
+            );
+
+            let sourcemap = SourceMap.generateEmptyMap({
+              projectRoot: pluginOptions.projectRoot,
+              sourceName,
+              sourceContent: code,
+            });
+
+            // This shouldn't be necessary probably a bug in the sourcemap library?
```

Comment:
Did you investigate this?

Reply:
Oh yeah I fixed that, forgot to remove the line